doc/ZserioUserGuide.md

```diff
@@ -137,7 +137,7 @@ extension for Zserio data inspection is not supported by Java API. By default is
 
 **`-withGrpcCode|-withoutGrpcCode`**
 
-Enables/disables generation of code for [GPRC](https://grpc.io/) services.
+Enables/disables generation of code for [GPRC](https://grpc.io/) services. It is enabled by default. 
```

Comment:
All other arguments have the sentence in form: "By default is enabled/disabled." Isn't it nicer to keep the same "format"?